app/frontend/src/lib/ui/components/filterGroup.test.js

```diff
@@ -311,4 +318,56 @@ describe('filterGroup', () => {
       expect(document.getElementsByClassName('govuk-accordion__section--expanded').length).toEqual(3);
     });
   });
+
+  describe('addUpdateOpenOrCloseEvent', () => {
+    test('displays close all when all elements are open', () => {
+      document.body.innerHTML = `
+      <form data-auto-submit="true">
```

Comment:
all this markup seems overly complex and unreadable for a test and i would be tempted to use `const XXX_SELECTOR = 'govuk-accordion__section-button` type approach mentioned in previous PR to help with this. Also the `closest` comment above would help this being simplified.